reloader/reloader.py

```diff
@@ -23,6 +34,10 @@ def dprint(*args, fill=None, fill_width=60, **kwargs):
 # check the link for comments
 # https://github.com/divmain/GitSavvy/blob/599ba3cdb539875568a96a53fafb033b01708a67/common/util/reload.py
 def reload_package(pkg_name, dummy=True, verbose=True):
+    if is_dependency(pkg_name):
+        reload_dependency(pkg_name)
```

Comment:
The `verbose` argument needs to be passed to `reload_dependency` and the further `reload_package`.

Also, we need to run `load_dummy` if `dummy` is true

```suggestion
if is_dependency(pkg_name):
    reload_dependency(pkg_name, verbose)
    if dummy:
	    load_dummy(verbose)
    return
```

Reply:
Makes sense.

Why is `dummy` needed? The doc comment says it's “needed to ensure TextCommand's and WindowCommand's are ready”, but I don't see why.

Reply:
We only need to run the dummy code once at the very end.

Reply:
So it means, dummy should be false in reload_dependency

Reply:
Fixed.